extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-extensions.js

```diff
@@ -88,6 +121,35 @@ export class QwcExtensions extends observeState(LitElement) {
             `;
     }
 
+    _favourite(e){
+        let favourites = this._getStoredFavourites();
+        let extName = e.detail.name;
+        if(favourites.includes(extName)){
+            const index = favourites.indexOf(extName);
+            if (index > -1) {
+                favourites.splice(index, 1);
+            }
+        }else{
+            favourites.push(extName);
+        }
+        this._setStoredFavourites(favourites);
+
+        this._favourites = this._getStoredFavourites();
+    }
+
+    _getStoredFavourites(){
+        let favourites = this.storageControler.get('favourites');
```

Comment:
What's the key exactly for that? Because I could see how you would want different favorites for different applications and my guess is that any application on the same host/port will use the same storage key? Or am I mistaken?

If so, I wonder if we should add the groupId:artifactId in the key?

Reply:
I did not want to add in the application name, as I again assumed your favs span applications. Like if you usually use swagger you want that for all applications. Not sure what is really the right answer, but this was simpler :)

Reply:
Also: none of the other things we store is per application. Example, we remember the state (open/close) and height if open of the footer. That span all applications too

Reply:
w.d.y.t ?